eZ/Publish/API/Repository/Tests/SearchServiceFulltextTest.php

```diff
@@ -194,21 +304,45 @@ public function testFulltextContentSearch($searchString, array $expectedKeys, ar
     }
 
     /**
-     * Test for the findLocations() method.
+     * Test for the findLocations() method on Solr 6.
      *
      * @param $searchString
      * @param array $expectedKeys
      * @param array $idMap
      *
      * @depends testPrepareContent
-     * @dataProvider providerForTestFulltextSearch
+     * @dataProvider providerForTestFulltextSearchSolr6
      */
-    public function testFulltextLocationSearch($searchString, array $expectedKeys, array $idMap)
+    public function testFulltextLocationSearchSolr6($searchString, array $expectedKeys, array $idMap): void
     {
-        if (($solrVersion = getenv('SOLR_VERSION')) && $solrVersion < 6) {
-            $this->markTestSkipped('Solr 4 detected, skipping as scoring won\'t match');
+        if (($solrVersion = getenv('SOLR_VERSION')) >= 7) {
```

Comment:
ATM This change might break tests for solr bundle 1.5 (supported on 1.xLTS) with Solr 4 if it pulls in kernel 7.5, but that is about to be fixed here: https://github.com/ezsystems/ezplatform-solr-search-engine/pull/150/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R15

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/2775/commits/a3fd4eabc5c1941f76fbf27f46edfe17493cda48